packages/app/lib/ui/widgets/note_field.dart

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import 'package:app/ui/widgets/card.dart';
+import 'package:app/ui/widgets/card_action_button.dart';
+import 'package:app/ui/widgets/card_header.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+typedef OnUpdate = void Function(String?);
+
+enum InputMode { read, edit }
+
+class NoteField extends StatefulWidget {
+  final String? value;
+  final OnUpdate onUpdate;
+
+  NoteField(this.value, {super.key, required this.onUpdate});
+
+  // @override
+  State<NoteField> createState() => _NoteFieldState();
+}
+
+class _NoteFieldState extends State<NoteField> {
+  InputMode _inputMode = InputMode.read;
+  late TextEditingController _textInputController;
+  late FocusNode _textInputFocusNode;
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  void initState() {
+    super.initState();
+    _textInputController = TextEditingController(text: widget.value ?? "");
+    _textInputFocusNode =
+        FocusNode(canRequestFocus: _inputMode == InputMode.edit);
+  }
+
+  @override
+  void didUpdateWidget(prev) {
+    super.didUpdateWidget(prev);
+    // Needed in order "reset" the initialValue of the input based on the change received by widget.value
+    _textInputController.text = widget.value ?? "";
+  }
+
+  @override
+  void dispose() {
+    _textInputController.dispose();
+    _textInputFocusNode.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final t = AppLocalizations.of(context)!;
+
+    final initialValue = widget.value ?? '';
+
+    return Form(
+        key: _formKey,
+        child: MeliCard(
```

Comment:
a lot of this is somewhat duplicated from the `EditableCard` but the difference in the header and some styles were enough for me to decide that it wasn't worth trying to update the implementation and instead just copy over some parts.

Reply:
Super, think it's good to not get too DRY to fast with these components